src/packages/cdk/src/app/types.ts

```diff
@@ -0,0 +1,51 @@
+import { ResponseCustomHeader } from 'aws-cdk-lib/aws-cloudfront';
+import { IVpc, SecurityGroup, InstanceType } from 'aws-cdk-lib/aws-ec2';
+import { PostgresEngineVersion } from 'aws-cdk-lib/aws-rds';
+
+export type GraphweaverAppConfig = {
+	name: string;
+	database: {
+		// Username for the database
+		username: string;
+		// Name of the database
+		name: string;
+		// Instance type for the database
+		instanceType?: InstanceType;
+		// Postgres version for the database
+		version?: PostgresEngineVersion;
+	};
+	adminUI: {
+		// Path to the admin UI build directory (relative to the project root)
+		buildPath: string;
+		// ARN of the certificate in AWS Certificate Manager to use for the website
```

Comment:
```suggestion
		// ARN of the certificate in AWS Certificate Manager to use for the website. Because CloudFront requires it, this certificate must be in Certificate Manager in the `us-east-1` region.
```